app/src/main/res/layout/activity_character_detail.xml

```diff
@@ -30,7 +31,8 @@
                 android:layout_height="wrap_content"
                 android:scaleType="centerCrop"
                 android:src="@drawable/character_detail_image"
-                app:layout_collapseMode="parallax" />
+                app:layout_collapseMode="parallax"
+                tools:ignore="ContentDescription" />
```

Comment:
add the content description instead of the ignore

Reply:
Done!